app/src/pages/Project.svelte

```diff
@@ -79,15 +76,18 @@
   }
 </style>
 
-{#await $project then result}
-  <ProjectSidebar />
+<DoubleSidebarLayout>
+  {#await $project then result}
+    <ProjectSidebar />
 
-  <div class="container">
-    <ProjectBreadcrumbs
-      style="position: fixed; top: 0;"
-      project={result.data.project} />
-    <div class="layout">
-      <Router {routes} />
+    <div class="container">
+      <NotificationFaucet />
```

Comment:
Shouldn't notifications be shown globally in the app?

Reply:
Indeed, this was my first stab at notifications. Improvements will go here: https://linear.app/issue/MON-34.